community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/PlannerName.scala

```diff
@@ -68,21 +73,21 @@ case object DPPlannerName extends CostBasedPlannerName {
  * Rule based planner for classes of queries where the cost based planner might end up with suboptimal plans.
  */
 case object FallbackPlannerName extends CostBasedPlannerName {
-  val name = "FALLBACK"
+  val name = CostBasedPlannerName.default.name
 }
 
 object PlannerName {
 
-  val default = FallbackPlannerName
-
   def apply(name: String): PlannerName = name.toUpperCase match {
-    case "RULE" => RulePlannerName
-    case "GREEDY" => GreedyPlannerName
-    case "IDP" => IDPPlannerName
-    case "DP" => DPPlannerName
-    case "FALLBACK" => FallbackPlannerName
+    case RulePlannerName.name => RulePlannerName
+    case GreedyPlannerName.name => GreedyPlannerName
+    case IDPPlannerName.name => IDPPlannerName
+    case DPPlannerName.name => DPPlannerName
+    case FallbackPlannerName.name => FallbackPlannerName
```

Comment:
We will never hit `FallbackPlannerName.name` since it is identical to `IDDPPlannerName.name`, is this intentional?